src/fragment-components/tiles/tile-morphism.tsx

```diff
@@ -0,0 +1,219 @@
+import { useFragment } from '../../context';
+
+/**
+ * Dropdown component to morph between different tiles
+ */
+export const TileMorphism = ({ component, setComponent }: any) => {
+	const tileTypes = [
+		{
+			id: 'tile',
+			text: 'Tile',
+			defaultComponent: {
+				type: 'tile',
+			}
+		},
+		{
+			id: 'clickabletile',
+			text: 'Clickable tile',
+			defaultComponent: {
+				type: 'clickabletile',
+			}
+		},
+		{
+			id: 'expandabletile',
+			text: 'Expandable tile',
+			defaultComponent: {
+				type: 'expandabletile',
+				expanded: true,
+				outline: false,
+				items: [
+					{
+						type: 'tilefold',
+						items: []
+					}
+				]
+			}
+		},
+		{
+			id: 'selectabletile',
+			text: 'Standalone selectable tile',
+			defaultComponent: {
+				type: 'selectabletile',
+				standalone: true,
+				disabled: false,
+				selected: false,
+			}
+		},
+		{
+			id: 'selectableTileGroup',
+			text: 'Multiselect tiles',
+			defaultComponent: {
+				type: 'selectableTileGroup',
+				tileGroup: true,
+			},
+			childDefaultComponent: {
+				type: 'selectabletile',
+				selected: false,
+				standalone: false,
+			}
+		},
+		{
+			id: 'radioTileGroup',
+			text: 'Radio tiles',
+			defaultComponent: {
+				type: 'radioTileGroup',
+				legend: 'Radio Tile Group',
+				tileGroup: true,
+			},
+			childDefaultComponent: {
+				type: 'radiotile',
+			}
+		}
+	];
+
+	const [fragment, setFragment] = useFragment();
+
+	const convert = ({ selectedItem }: any) => {
+		// Conversion to different types
+		if (component?.tileGroup && selectedItem.defaultComponent?.tileGroup) {
+			setComponent(
+				groupToGroup(selectedItem, component)
+			);
+		} else if (component?.tileGroup && !selectedItem.defaultComponent?.tileGroup) {
+			const parentComponent: any = getParentComponent(fragment.data, component);
+
+			setFragment({
+				...fragment,
+				data: updatedState(fragment.data, {
+					type: 'update',
+					component: groupToSingle(selectedItem, component, parentComponent),
+				})
+			});
+		} else if (!component.tileGroup && selectedItem.defaultComponent?.tileGroup) {
+			setComponent(
+				singleToGroup(selectedItem, component)
+			);
+		} else {
+			setComponent(
+				singleToSingle(selectedItem, component)
+			);
+		}
+	}
+
+	return <Dropdown
+		id="tile-morpher"
+		label="Convert to another tile type"
+		titleText="Tile type"
+		items={tileTypes.filter(({ id }) => id !== component.type)}
+		itemToString={(item: any) => (item ? item.text : '')}
+		onChange={convert}
+	/>
+}
+
+// Combine all items from 'top' & 'bottom' folds into a single array
+const getExpandableTileItems = (expandableTile: any) => {
+	// Destructuring existing items to prevent changing default object
+	const items = [...expandableTile.items];
+
+	// Find tileFold index & retrieve it from the list
+	const tileFoldIndex = items.findIndex((item: any) => item.type === 'tilefold');
+	const tileFoldItems = items.splice(tileFoldIndex, 1);
```

Comment:
This looks a bit similar to the getFoldObjects in expandable tiles, maybe have a shared utility function called getTileFoldItems? If you don't decide on the shared function, same suggestion from the getFoldObjects function applies here as well.